src/pushNotifications/PermissionHandling.ts

```diff
@@ -107,3 +111,26 @@ const showInitialPushAlert = (): void => {
     { cancelable: false }
   );
 };
+
+export const showPermissionRequiredAlert = (approveCallback: () => void) => {
+  const {
+    abort,
+    approve,
+    permissionMissingTitle,
+    permissionRequiredBody
+  } = texts.pushNotifications;
+
+  Alert.alert(permissionMissingTitle, permissionRequiredBody, [
+    {
+      text: approve,
+      onPress: async () => {
+        await setInAppPermission(true);
+        approveCallback();
+      }
+    },
+    {
+      text: abort,
+      style: 'cancel'
+    }
```

Comment:
this is the only alert without `{ cancelable: false });` now. intended?
and the order of "approve" and "cancel" is also different to the other alerts. there are cancel first and approve second.

Reply:
I removed the `{ cancelable: false }` from the `PermissionMissingAlert` as well now, as it is not required in either of these two, and moved the cancel button to be the first in the array.